Superlista/Components/ListCard.swift

```diff
@@ -140,17 +140,29 @@ struct ListCard: View {
                     }
                 }
                 .alert(isPresented: $showAlertDelete){
-                    Alert(title: Text("Remover \(dataService.currentList!.title)"), message: Text("DeleteListAlertText"), primaryButton: .cancel(), secondaryButton: .destructive(Text("DeleteListAlertButton"), action:{
-                        dataService.removeList(dataService.currentList!)
-                        showAlertDelete = false
+                    let title = (dataService.currentList != nil) ? dataService.currentList?.title : ""
+                    
+                    let text = "\(NSLocalizedString("Remover", comment: "Remover")) \(String(describing: title))?";
+                    
+                    return Alert(title: Text(text), message: Text("DeleteListAlertText"), primaryButton: .cancel(), secondaryButton: .destructive(Text("DeleteListAlertButton"), action: {
+                        if let currentList = dataService.currentList {
+                            dataService.removeList(currentList)
+                            showAlertDelete = false
+                        }
                     }))
                 }
                 
                 ZStack {}
                 .alert(isPresented: $showAlertDuplicate){
-                    Alert(title: Text("Duplicar \(dataService.currentList!.title)"), message: Text("DuplicateListAlertText"), primaryButton: .cancel(), secondaryButton: .default(Text("DuplicateListAlertButton"), action:{
-                        dataService.duplicateList(of: dataService.currentList!)
-                        showAlertDuplicate = false
+                    let title = (dataService.currentList != nil) ? dataService.currentList?.title : ""
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()